deployment_report/templates/viya_deployment_report.html.j2

```diff
@@ -335,61 +352,76 @@
           {% endif %}
         </div>
       </div>
-      {% endfor %}
-    </div>
+    {% endfor %}
+  {% endif %}
   </div>
-  {# Cluster Overview: Configmaps Accordion #}
+  </div>
+  {# Configurations: Configmaps Accordion #}
 
-  {# Cluster Overview: Secrets Accordion #}
-  {% set kind_name = "Secrets" %}
+  {# Configurations: Secrets Accordion #}
   <div class="jq-accordion">
-    <h2>Secrets</h2>
+  <h2>Secrets</h2>
+  {% set kind_name = "Secrets" %}
+  <div>
+  {% if report_data.kubernetes.secrets|length == 0 %}
     <div>
-      {% for type, info in report_data.kubernetes.secrets.items() %}
-      <div class="jq-accordion">
-        <h7>{{ type }}</h7>
-        <div>
-          {% set ns = namespace() %}
-          {% for key, value in info.items() %}
-            {% set ns.keyrefurl = '' %}
-            {% set ns.keylist = '' %}
-            {% for key2, value2 in value.items() %}
-              {% if key2 == "name" %}
-                {% set ns.keylist = ns.keylist ~ "<tr><th>Keys</th></tr>" %}
-                {% for i in value2 %}
-                  {% set ns.keylist = ns.keylist ~ "<tr><td>" ~ i ~ "</th></tr>" %}
-                {% endfor %}
-              {% else %}
-                {% set mybr = '</br>' %}
-                {% set ns.secretvalue = '' %}
-                {% for i in value2 %}
-                  {% set ns.secretvalue = ns.secretvalue ~ '<a class="nested-accordions-rel-link" href="#pod-secrets_' ~ i ~ '">' ~ mybr ~ '&bull; ' ~ i ~ '</a>' %}
-                {% endfor %}
-                {% if ns.secretvalue != '' %}
-                  {% set ns.keyrefurl = ns.keyrefurl ~ mybr ~ "Referenced pods containers:" ~ ns.secretvalue ~ mybr ~ mybr %}
+      <div class="notification-box info-box">
+        <i class="fa fa-info-circle notification-box-icon" aria-hidden="true"></i>
+        <div class="notification-box-text margin-block-end-1em">
+          No secrets were found in namespace [{{ report_data.kubernetes.namespace }}].
```

Comment:
Let's change `secrets` to `Secrets` to better align with how k8s formats the name.